### PR TITLE
CI to copy config to `gs://cpg-config/templates`

### PR DESCRIPTION
**Pull request by vladsavelyev**

### CODE REVIEW

**Review by illusional — APPROVED** [review]
Do we need to give the image deployer write access to that bucket, or will it already have it.

Also noticed the CI is broken for this repo too

File: .github/workflows/toml-changed.yaml
Comment (lgruen) [inline .github/workflows/toml-changed.yaml:86]:
```suggestion
          gcloud storage cp images.toml gs://cpg-config/templates/images/images.toml
```

**Review by lgruen — APPROVED** [review]

**Comment by vladsavelyev** [comment]
> Also noticed the CI is broken for this repo too

CI checks the differences in images.toml between the previous revision and the current revision. That's why I split my previous changes into 2 stacked PRs: one adds images.toml, another one adds CI that checks the differences. You accidentally merged the second one into the first one, so after we merged them altogether, it broke the first CI run :) But consequent merges should be alright.

**Comment by vladsavelyev** [comment]
> Do we need to give the image deployer write access to that bucket, or will it already have it.

Yeah I think we need to do that, good spot

**Comment by vladsavelyev** [comment]
We probably also want to read infra-specific parameters like `PREFIX: gs://cpg-config/templates`, `PREFIX: australia-southeast1-docker.pkg.dev/cpg-common/images` from somewhere, e.g. secrets deployed in Pulumi. What do you think @illusional @lgruen

**Comment by lgruen** [comment]
I think that would be handy, as it could be used both by this CI (to write blobs) and the analysis-runner (to read blobs).

**Comment by vladsavelyev** [comment]
@lgruen @illusional could you please point me to where the service account used to deploy images here is created?

**Pull request merged**
